src/lsif.ts

```diff
@@ -0,0 +1,1537 @@
+/**
```

Comment:
You can [create a `.gitattributes` file](https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github) so that this is ignored in the future, even if there are small changes (right now, GitHub hid the changes because there was a large diff).